tests/entrypoints/openai/test_chat_with_tool_reasoning.py

```diff
@@ -6,6 +6,10 @@
 
 from ...utils import RemoteOpenAIServer
 
+pytestmark = pytest.mark.skip(reason="Currently, these test cases are  \
```

Comment:
can you update .buildkite/test-pipeline.yaml L121 to ignore this file instead of this.

this will ignore the tests when users run `pytest` locally.